SPECS/filesystem/filesystem.spec

```diff
@@ -36,6 +36,7 @@ install -vdm 755 %{buildroot}/media
 install -dv -m 0750 %{buildroot}/root
 install -dv -m 1777 %{buildroot}/tmp %{buildroot}/var/tmp
 install -vdm 755 %{buildroot}/usr/{,local/}{bin,include,lib,sbin,src}
+install -vdm 755 %{buildroot}/usr/src/mariner
```

Comment:
> is /usr/src/"mariner" the right thing here? there is rebranding afoot for "azurelinux"

The only reason to leave this is in case there is any end users who have hardcoded this path, and are expecting to build packages in this path.

Otherwise, we don't need the fallback behavior (of root still using /usr/src/mariner) and we should simply use standard topdir location that rpm has provided for over 16 years.

Personally I would *much* rather drop the fallback behavior and go with the rpm standard.

Reply:
ah i see @PawelWMS is already planning to rename the location so there's no need to keep backwards compatibility; I'll update the PR to completely move to the standard then (i.e. remove our %_topdir override completely)

Reply:
latest push has our %_topdir override removed completely